warp-element/src/utils.js

```diff
@@ -1,27 +1,32 @@
 import { unsafeCSS } from 'lit';
 
-export const isServer = () => {
+const isServer = () => {
     return !(typeof window !== 'undefined');
 };
 
-export const parseBrand = (str = '') => {
+const parseBrand = (str = '') => {
     if (str.toLocaleLowerCase().includes('blocket')) return { sld: 'blocket', tld: 'se' };
     if (str.toLocaleLowerCase().includes('tori')) return { sld: 'tori', tld: 'fi' };
     if (str.toLocaleLowerCase().includes('finn')) return { sld: 'finn', tld: 'no' };
     if (str.toLocaleLowerCase().includes('dba')) return { sld: 'dba', tld: 'dk' };
     return { sld: 'finn', tld: 'no' };
 };
 
+
+/*
+ * @param {string} [brandStr] - a string with brand information
```

Comment:
Should this be "an URL" or something like that?

Reply:
I think string is correct